cpanfile

```diff
@@ -31,13 +31,15 @@ requires 'Ref::Util';
 requires 'Try::Tiny';
 requires 'Type::Params';
 requires 'Types::URI';
+requires 'feature';
```

Comment:
I think rather than adding `feature` here, we can actually just remove the use of `feature` from the dist.  It's not bringing much to the table.